backend/globaleaks/models/__init__.py

```diff
@@ -824,6 +824,22 @@ class ApplicationData(Model):
     fields = JSON()
 
 
+class Log(BaseModel):
+    """
+    Note -- is BaseModel because Log.id is Integer
+    """
+    id = Int(primary=True)
+    code = Unicode()
+    args = JSON()
+    log_date = DateTime()
+    # This is the identification string, is composed by a 'admin' or 'receiver_$UUIDv4'
+    subject = Unicode()
+    log_message = Unicode()
+    log_level = Int()
+    mail = Bool()
+    mail_sent = Bool()
+
```

Comment:
Please write here the defaults; they are useful to avoid explicit initialization if not needed and help also who read the code in understanding their usage.